pyproject.toml

```diff
@@ -40,6 +40,7 @@ azure-identity = {version = "^1.15.0", optional = true}
 azure-keyvault-secrets = {version = "^4.8.0", optional = true}
 google-cloud-kms = {version = "^2.21.3", optional = true}
 resend = {version = "^0.8.0", optional = true}
+sentence-transformers = {version = "^2.7.0", optional = true}
```

Comment:
@powerhouseofthecell this is a fairly large dep. I'm not sure we want to introduce sentence-transformers as a proxy dependency without good reason. 

Sentence transformers (in my experience) also doesn't work well on cpu servers

requiring me to host this on a GPU. 



In that situation, does it even make sense to have it be on the same machine as the proxy vs. a separate API endpoint we can just interact with?

Reply:
ah got it - in that case i'll switch to having it use a networked embeddings model and just recommend that this model be used.

i'll remove that dependency

Reply:
Thanks! 

Does this behave well with text-embeddings-inference? 

Not sure if sentence-transformers has a known API we can interact with

Reply:
@krrishdholakia - removed that dependency, thank you!